src/_help/customization-options/logo-meta-images.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Logo and Meta images
+---
+
+- TOC
+{:toc}
+
+## Logo and favicon
+
+You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.
+
+From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.
+
+![](/images/help/logo-favicon.png)
+
+To provide the best image quality, it is recommended to adhere to the following dimensions for your logo: 140px*36px.
```

Comment:
Are we sure of these dimensions (cc @thimy)? Also, should we mention that our renderer will try to adapt the best it can if other dimensions are provided?

Reply:
I've checked with @Thimy for these dimensions but I'll wait for her confirmation.
I didn't know that our renderer could do this, adding that to the next commit.